website/content/partials/known-issues/database-skip-static-role-rotation.mdx

```diff
@@ -14,4 +14,5 @@ their passwords rotated when Vault is restarted. The password rotation will not
 occur on restart if the static role has already had its password rotated.
 
 #### Workaround
-Currently, there is no workaround except to avoid a restart of the backend.
+Currently, there is no workaround except to avoid a restart of the backend. Or 
+upgrade to 1.19.3+
```

Comment:
```suggestion
Upgrade to 1.19.3+. For versions 1.19.0, 1.19.1, and 1.19.2, the only
workaround is to avoid restarting the backend.
```
We should list the preferred solution (upgrading) first.